WinPath/src/Backup.cs

```diff
@@ -81,27 +84,42 @@ public class Backup
                 case HandleEventType.ListAllBackups:
                     Console.WriteLine("User Backups:");
                     Console.WriteLine("Filename" + spaces + " | Date of creation");
-                    Console.WriteLine(seperator);
-
-                    foreach (FileInfo backupFile in userDirFileInfo)
-                        Console.WriteLine(
-                            backupFile.Name
-                                + " | "
-                                + (
-                                    long.TryParse(
-                                        backupFile.Name,
-                                        out temp
-                                    )
-                                        ? DateTime.FromFileTime(temp)
-                                        : "<Parsing error>"
-                                  )
-                        );
-
-                    Console.WriteLine(seperator + Console.Out.NewLine);
+
+                    var strings = new string[userDirFileInfo.Length + 1];
+                    var len = 0;
+
+                    for (var i = 0; i < userDirFileInfo.Length; i++)
+                    {
+                        var backupFile = userDirFileInfo[i];
+                        var status = long.TryParse(backupFile.Name, out temp);
+                        var str = backupFile.Name + " | " + (status
+                            ? DateTime.FromFileTime(temp).ToString(CultureInfo.InvariantCulture)
```

Comment:
Hello, can you possibly remove that `.ToString(CultureInfo.InvariantCulture)` part, I prefer to not restrict that date time to a specific format, for all regions. Can you possibly remove that. Thanks.